### PR TITLE
Add community meetings table 

### PR DESCRIPTION
**Pull request by chandrashritii**
**Description**

This PR adds a table with details of all the community meetings on the layer5/community page.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Jashpatel1** [comment]
Responsivenesss in mobile and iPad view
image

File: _includes/community/meshmates-program-summary.html [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
+  <td><a href = "https://layer5io.slack.com/archives/C011VPREG2Z">#meshery-ci</a></td>
+  <td><a href = "https://bit.ly/3aXOaAs ">Minutes</a></td>
+  <td><a href = "https://bit.ly/2QocW3s">Zoom Link</a></td>
+  <td><a href = "https://www.youtube.com/watch?v=ds9D2KgZKxo&list=PL3A-A6hPO2IM7rYiKxG4l3eQNc6X3IUex">Youtube</a></td>
+</tr>
+<tr>
+  <td>Newcomers' Meeting</td>
+  <td><a href = "https://layer5io.slack.com/archives/C019426UBNY">#newcomers</a></td>
+  <td><a href = "https://bit.ly/353Aa7D">Minutes</a></td>
+  <td><a href = "https://bit.ly/34SFIlc">Zoom Link</a></td>
```
Comment (leecalcote) [inline _includes/community/meshmates-program-summary.html:43]: Replace Zoom links with `https://meet.layer5.io` URLs.

**Comment by chandrashritii** [comment]
That's funny. I'm getting a different view on my browser. Thanks for pointing that out @Jashpatel1. Which browser and OS are you using?

Screenshot_2020-09-14 Community

File: _includes/community/meshmates-program-summary.html [hunk 2]
```diff
@@ -1,26 +1,54 @@
...
+  <td><a href = "https://www.youtube.com/watch?v=E1juH6Rjovo&list=PL3A-A6hPO2INVEknQ4R-VwDMUT9yM_FSm">Youtube</a></td>
+</tr>
+<tr>
+  <td>SMI Community Meeting</td>
+  <td><a href = "https://layer5io.slack.com/archives/C010H0HE2E6">#smi</a></td>
+  <td><a href ="https://docs.google.com/document/d/1NTBaJf6LhUBlF8_lfvBBt_MbyPvT-6CZNg6Ckpm_yCo/edit">Minutes</a></td>
+  <td><a href = "https://zoom.us/my/cncfsmiproject"> Zoom Link </a></td>
```
Comment (leecalcote) [inline _includes/community/meshmates-program-summary.html:50]: If we're going to list the SMI Community Meeting, let's go ahead and list the CNCF SIG Network meeting as well.

Both of these two meetings should be colorized or marked differently somehow to show that both of them are CNCF meetings.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: _includes/community/community-members.html [hunk 3]
```diff
@@ -1,29 +1,49 @@
...
+  advancing our open source initiatives.
   <p class="indent">
-    We believe that all contributors should expect and be part of a safe and
-    friendly environment for constructive contribution. We can more effectively
-    and successfully compare and challenge different ideas to find the best
-    solutions for advancement, while building the size, diversity, and strength
-    of our community.
-  </p>
-  <div width="100%" style="justify-content:center; text-align: center;">
+    We believe that every contributor deserves a safe and friendly environment, enabling them to effectively compare different ideas and find the best solutions for advancement, while building the size, diversity, and strength of the community.
```
Comment (leecalcote) [inline _includes/community/community-members.html:11]: We'll want to enforce a column wrap of 80.

You can use VS Code's "Prettier - Code formatter" plugin to achieve this.

File: _includes/community/meshmates-program-summary.html [hunk 4]
```diff
@@ -1,26 +1,54 @@
...
-  <h6>An onboarding and mentoring program</h6>
-  <p class="indent">
-    The Layer5 community is growing at a tremendous rate. No matter the size, we know that connection is what makes the Layer5 community different from any other community you join. Whether it is your first Layer5 meeting or your tenth contribution, as community organizers, we want to make sure you have a great time and are able to take advantage of all of the 
-    amazing learning, contributing, and community that Layer5 has to offer. 
-  </p>  
-  <p class="indent">
-    In the Layer5 mentor program, we will match you with an 
-    awesome MeshMate so you have a connection and guide from the start.
-  </p>
+<table>
```
Comment (leecalcote) [inline _includes/community/meshmates-program-summary.html:3]: This file is for the meshmates-program-summary and should contain that summary. A new file can be created to capture the table of community workgroup resources.

File: assets/css/materialize.css [hunk 5]
```diff
@@ -10741,3 +10741,25 @@ div.slack {
     font-size: 8px;
   }
 }
+
+/*Community Meetings Table */
+table {
```
Comment (leecalcote) [inline assets/css/materialize.css:10746]: Redefining these styles in this way will have a site-wide affect. Namespace these styles or use selectors.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by Jashpatel1** [comment]
> That's funny. I'm getting a different view on my browser. Thanks for pointing that out @Jashpatel1. Which browser and OS are you using?
> 
> Screenshot_2020-09-14 Community

Browser: Google Chrome OS: Windows, Can @leecalcote and @Nikhil-Ladha check this out?

**Comment by Nikhil-Ladha** [comment]
The same is happening with me on mobile device. @chandrashritii @Jashpatel1 . I haven't checked in browser yet, will do that too.

**Review by Jashpatel1 — DISMISSED** [review]

**Review by leecalcote — CHANGES_REQUESTED** [review]
Screen Shot 2020-09-14 at 12 54 12 PM

- Will you move the table to the bottom of the page and have it span both columns?
- Will you move the Layer5 MeshMates section to its original position?
- Will you list the Layer5 meeting in a high contrast color (e.g. black) and the CNCF meetings in the low contrast color (not green)?
- Will you change the table column headings background color to a color other than the two greens using in the Layer5 logo?

**Comment by chandrashritii** [comment]
@leecalcote Using #20BF9F as the column heading color here. Is there a more appropriate color that can be used?

**Comment by chandrashritii** [comment]
@leecalcote @Jashpatel1 
Some other possible table column heading colors, sourcing from the Layer5 brand colors:

Screenshot_2020-09-15 Community Screenshot_2020-09-15 Community(1) Screenshot_2020-09-15 Community(2)

File: _includes/community/community-meetings-table.html [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+<table id="table101">
+<tr>
+  <th>Group</th>
+  <th>Slack Channel</th>
+  <th>Meeting Minutes</th> 
+  <th>Meeting Link</th>
+  <th>Meeting Recordings</th>
+</tr>
+<tr>
+  <td><b><font color = #000000>Layer5 Community Meeting</font></b></td>
```
Comment (leecalcote) [inline _includes/community/community-meetings-table.html:14]: Avoid inline style when you can. Use quotation marks around color. `<b>` and `<font>` are deprecated tags. Use `class=black-text`.

File: _includes/community/community-meetings-table.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<div align="center"><h3 class="black-text">Layer5 Meetings</h3></div>
+
+<div style="overflow-x:auto;">
+<table id="table101">
```
Comment (leecalcote) [inline _includes/community/community-meetings-table.html:5]: Rename ID.

File: _includes/community/community-meetings-table.html (hunk 6 above)
Comment (leecalcote) [reply _includes/community/community-meetings-table.html:14]: Remove emphasis (use normal font weight) for all values in this column.

File: _includes/community/community-meetings-table.html [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+  <td><b><font color = #000000>Layer5 Community Meeting</font></b></td>
+  <td><a href = "https://layer5io.slack.com/archives/CDM0ACDM5">#general</a></td>
+  <td><a href="https://bit.ly/2It7IA4">Minutes</a></td>
+  <td><a href="https://meet.layer5.io/community">Zoom Link</a></td>
+  <td><a href = "https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">Youtube</a></td>
+</tr>
+<tr>
+  <td><b><font color = #000000>Websites Meeting</font></b></td>
+  <td><a href = "https://layer5io.slack.com/archives/C015QJKUMPU">#websites</a></td>
+  <td><a href = "https://bit.ly/2XK4eQV"> Minutes</a></td>
```
Comment (leecalcote) [inline _includes/community/community-meetings-table.html:23]: Remove extraneous whitespace in " Minutes" hyperlink.

Remove whitespace between `href = "https..."`. Use `href="https..."`.

File: _includes/community/community-members.html [hunk 9]
```diff
@@ -1,29 +1,25 @@
...
-  <div width="100%" style="justify-content:center; text-align: center;">
+  We believe that every contributor deserves a safe and friendly environment, enabling them to effectively compare different ideas and find the best solutions for advancement, while building the size, diversity, and strength of the community.
+  </p>    
+  <div width="100%" style="justify-content:center; text-align: left;">
     <h3 style="text-align: center; color: aliceblue;display:inline-flex;
     ">
       <a
         style="
-          font-size: 0.65em;
+          font-size: 0.60em;
```
Comment (leecalcote) [inline _includes/community/community-members.html:18]: Revert this size change.

Leave these links center-aligned.

Comment (leecalcote) [reply _includes/community/community-members.html:18]: Is the `div` needed?

File: _includes/community/meshmates-program-summary.html [hunk 10]
```diff
@@ -1,22 +1,20 @@
...
+  <p class="indent">  
+    In the Layer5 mentor program, we will match you up with a <font color:  aliceblue>MeshMate</font> to support and guide you along the way, by enabling you to identify your area of interests within the projects, directing you towards working groups to join, growing your Cloud Native knowledge, and establishing a network of relationships.
+  </p>
+  <div width="100%" style="justify-content:center; text-align: left;">
     <h3 style="text-align: center; color: aliceblue;display:inline-flex;
     ">
       <a
         style="
-          font-size: 0.65em;
+          font-size: 0.58em;
```
Comment (leecalcote) [inline _includes/community/meshmates-program-summary.html:17]: Revert this size change.

Leave these links center-aligned.

File: assets/css/materialize.css [hunk 11]
```diff
@@ -10741,3 +10741,26 @@ div.slack {
...
+#table101 td {
+  border: 1px solid #dcdcdc;
+  text-align: left;
+  padding: 8px;
+  column-gap: 80px;
+}
+
+#table101 th {
+  background-color: #20BF9F;
+  text
```
Comment (leecalcote) [inline assets/css/materialize.css:10761]: Remove `text`

File: assets/css/materialize.css [hunk 12]
```diff
@@ -10741,3 +10741,26 @@ div.slack {
...
+
+#table101 td {
+  border: 1px solid #dcdcdc;
+  text-align: left;
+  padding: 8px;
+  column-gap: 80px;
+}
+
+#table101 th {
+  background-color: #20BF9F;
```
Comment (leecalcote) [inline assets/css/materialize.css:10760]: Use "#477E96;"

Use bold font weight for column headings.

Use white for font color.

Set border-radius to 0px.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by Jashpatel1** [comment]
> @leecalcote @Jashpatel1
> Some other possible table column heading colors, sourcing from the Layer5 brand colors. Are any of these better suited?

I think the current green serves good for the purpose of heading row.

**Comment by Nikhil-Ladha** [comment]
>  Screenshot_2020-09-15 Community(2)

This one looks good to me

**Review by leecalcote — CHANGES_REQUESTED** [review]
There is an unfortunate scrollbar showing. Could you have a look at this?

Screen Shot 2020-09-16 at 1 49 29 PM

File: _includes/community/meshmates-program-summary.html (hunk 1 above)
Comment (leecalcote) [reply _includes/community/meshmates-program-summary.html]: These links have not been updated.

File: _includes/community/community-members.html (hunk 9 above)
Comment (leecalcote) [reply _includes/community/community-members.html]: Was the font size reverted?

File: _includes/community/community-members.html (hunk 3 above)
Comment (leecalcote) [reply _includes/community/community-members.html]: It's not critical, but text isn't wrapped here.

File: assets/css/materialize.css (hunk 11 above)
Comment (leecalcote) [reply assets/css/materialize.css]: `Text` needed?

File: assets/css/materialize.css (hunk 5 above)
Comment (leecalcote) [reply assets/css/materialize.css]: Should not change global style.

File: _includes/community/community-meetings-table.html [hunk 13]
```diff
@@ -0,0 +1,64 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<div align="center"><h3 class="black-text">Layer5 Meetings</h3></div>
+
+<div style="overflow-x:auto;">
+<table id="table101">
+<tr>
+  <th>Group</th>
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply _includes/community/community-meetings-table.html:7]: Still needs right-aligned and column renamed.

File: _includes/community/community-members.html (hunk 9 above)
Comment (chandrashritii) [reply _includes/community/community-members.html]: Yes. The font size was reverted.

**Comment by chandrashritii** [comment]
@leecalcote The scrollbar has been removed, along with the other changes.

**Review by leecalcote — APPROVED** [review]
Thank you, @chandrashritii

**Comment by Nikhil-Ladha** [comment]
Screenshot_20200918-093727__01
One issue with the table in mobile view.

**Pull request merged**